spec/support/capybara.rb

```diff
@@ -0,0 +1,12 @@
+Capybara.register_driver :headless_chrome do |app|
```

Comment:
Capybara has a built-in headless chrome driver, but it doesn't use enough flags to satisfy Travis-CI or WSL2 as they don't have `$DISPLAY`. This setup really makes it headless.